cpp/include/raft/stats/mean.cuh

```diff
@@ -90,13 +53,13 @@ void mean(Type *mu, const Type *data, IdxType D, IdxType N, bool sample,
     dim3 grid(raft::ceildiv(N, (IdxType)RowsPerBlk),
               raft::ceildiv(D, (IdxType)ColsPerBlk));
     CUDA_CHECK(cudaMemsetAsync(mu, 0, sizeof(Type) * D, stream));
-    meanKernelRowMajor<Type, IdxType, TPB, ColsPerBlk>
+    detail::meanKernelRowMajor<Type, IdxType, TPB, ColsPerBlk>
```

Comment:
Same here wrt invoking kernels from public API. Just proxy to the function in the details API.